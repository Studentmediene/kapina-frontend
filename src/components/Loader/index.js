import React from 'react';
import { ScaleLoader } from 'react-spinners';
import styles from './styles.css';

class Loader extends React.Component {
  render() {
    return (
      <div className={styles.loading}>
        <ScaleLoader color={'#ECB61C'} />
      </div>
    );
  }
}

export default Loader;
