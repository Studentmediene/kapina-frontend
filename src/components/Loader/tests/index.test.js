import React from 'react';
import { shallow } from 'enzyme';

import Loader from '../';

describe('<Loader />', () => {
  it('renders correctly', () => {
    const tree = shallow(<Loader />);
    expect(tree).toMatchSnapshot();
  });
});
