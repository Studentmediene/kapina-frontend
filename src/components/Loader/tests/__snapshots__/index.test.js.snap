// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> renders correctly 1`] = `
<div
  className="loading"
>
  <onlyUpdateForKeys(Loader)
    color="#ECB61C"
    height={35}
    loading={true}
    margin="2px"
    radius={2}
    width={4}
  />
</div>
`;
